scripts/rollup/build.js

```diff
@@ -402,8 +400,8 @@ function getPlugins(
       forbidFBJSImports(),
       // Use Node resolution mechanism.
       resolve({
-        // skip: externals, // TODO: options.skip was removed in @rollup/plugin-node-resolve 3.0.0
-        preferBuiltins: bundle.preferBuiltins,
```

Comment:
Started this PR before `bundle.preferBuiltins` was added. That's also a valid API (but should probably be `bundle.targetsNodeJS` or something).